### DISCUSSION TITLE
Update time

### DISCUSSION THREAD

**Discussion by BL1Z33** [anuraghazra/github-readme-stats#2764] [Q&A] [answered]
How fast does the top langs card update?

**Comment by qwerty541** [comment, marked as answer]
Hey, @rivalryxd! It depends from `cache_seconds` option. By default it was set to 14400 seconds ( 4 hours ). You can set custom value from 14400 to 86400 seconds. See: https://github.com/anuraghazra/github-readme-stats#common-options

**Reply by BL1Z33** [reply]
> Hey, @rivalryxd! It depends from `cache_seconds` option. By default it was set to 14400 seconds ( 4 hours ). You can set custom value from 14400 to 86400 seconds. See: https://github.com/anuraghazra/github-readme-stats#common-options

Ok, thanks a lot!
